### ISSUE TITLE
Manually enter the "Root of the repository"

### ISSUE THREAD

**Issue by djelom** [SRombauts/UE4GitPlugin#75]
As discused in [this thread](https://answers.unrealengine.com/questions/805501/source-control-how-to-specify-the-repository-that.html) on the forums, we are using several sub repos in the /Content/ folder. 
Those sub repos are ignored from the main repo.
That all works well in unreal and also in git, however I lost the ability to diff the changes in those sub repos. 

I suspect that I was able to define the "root of the ropository" to be one of those sub repos it'd magically work?

There might be complications involved with the feature, but it would be super nice to be able to hop from one repo to another, even if that involves manually writing the path of the repo.

**Comment by SRombauts** [comment]
Yes sure, but as I said previously, this is not a priority for me right now, and it adds quite a bit of complexity

**Comment by ykaganov** [comment]
@SRombauts 
I'm trying to add this feature myself, however when editing SGitSourceControlSettings.cpp upon build my changes seem to be deleted unless I'm imaging things ;)). It seems that this file is overridden by the version from the Unreal source code. Could you please advise? Is this something that happens by design? Wrong configuration on my part?
Also, I would be happy to hear more about the complications that I'm going to encounter if you thought about that...
Thanks in advance!

Yan

**Comment by ykaganov** [comment]
@SRombauts 
I Created a pull request with this feature.
https://github.com/SRombauts/UE4GitPlugin/pull/138
